src/QZXing.cpp

```diff
@@ -398,7 +430,9 @@ QString QZXing::decodeImage(const QImage &image, int maxWidth, int maxHeight, bo
                 }
 
                 emit tagFound(string);
-                emit tagFoundAdvanced(string, foundedFmt, charSet_);
+
+                QVariantMap metadataMap = metadataToMap(res->getMetadata());
+                emit tagFoundAdvanced(string, foundedFmt, charSet_, metadataMap);
```

Comment:
To avoid braking backward compatibility it would be better to emit both overloads of tagFoundAdvanced: 

```cpp
QVariantMap metadataMap = metadataToMap(res->getMetadata());
emit tagFoundAdvanced(string, foundedFmt, charSet_, metadataMap);
emit tagFoundAdvanced(string, foundedFmt, charSet_);
```